src/arch/amdgpu/vega/insts/ds.cc

```diff
@@ -4925,5 +4925,492 @@ namespace VegaISA
         vdst2.write();
         vdst3.write();
     } // completeAcc
+    // --- Inst_DS__DS_READ_B64_TR_B4 class methods ---
+
+    Inst_DS__DS_READ_B64_TR_B4::Inst_DS__DS_READ_B64_TR_B4(InFmt_DS *iFmt)
+        : Inst_DS(iFmt, "ds_read_b64_tr_b4")
+    {
+        setFlag(Load);
+    } // Inst_DS__DS_READ_B64_TR_B4
+
+    Inst_DS__DS_READ_B64_TR_B4::~Inst_DS__DS_READ_B64_TR_B4()
+    {
+    } // ~Inst_DS__DS_READ_B64_TR_B4
+
+    void
```

Comment:
previously these instructions had short descriptions of what they are doing above.  Is it possible to bring that back?

Reply:
gotcha